### DISCUSSION TITLE
OnChange/OnUpdate event after making some changes to editor's content

### DISCUSSION THREAD

**Discussion by Koffer0-0** [Milkdown/milkdown#1541] [Ideas]
Is it possible to extend the functionality of Milkdown Crepe editor to trigger OnChange or OnUpdate events whenever the editor's content is modified?

This would be especially useful in scenarios where the editor is part of a Vue.js component, and you need to update the content in real-time as the user types or makes changes. It would allow seamless synchronization between the editor's state and the component’s data.

Expected Behavior
An event like OnChange or OnUpdate should be fired whenever the content of the editor is modified. This event should be accessible from the outside to allow components to listen to and react to content changes in real-time.
Actual Behavior
Currently, there doesn’t appear to be a built-in way to capture and respond to content changes directly via such events in Milkdown Crepe.

Additional Context
The OnChange or OnUpdate event would help in cases where the editor is used for real-time content updates, ensuring synchronization between the editor and external states like component data or server updates.

**Comment by jclaveau** [comment]
I guess your are looking for the 'listener' plugin https://milkdown.dev/docs/api/plugin-listener ?
